Integrations/CiscoFirepower/CiscoFirepower.py

```diff
@@ -1273,7 +1260,8 @@ def main():  # pragma: no cover
     LOG('command is %s' % (demisto.command(),))
     try:
         if demisto.command() == 'test-module':
-            return_outputs(*test_module_command())
+            return_outputs('ok')
+            # Login is performed at the beginning of each flow (line 1271) if the login fails we return an error.
```

Comment:
login changed to line 1258 (Not recommending to write line numbers it could change many times)